Car-slider.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <style>
+      * {
+        box-sizing: border-box;
+        margin: 0;
+        padding: 0;
+        outline: none;
+      }
+
+      .card {
+        display: inline-block;
+        background-color: rgb(248, 246, 246);
+        border-radius: 5px;
+        border: 1px solid rgb(202, 202, 202);
+        margin:3px;
+      }
+
+      .card__carousel {
+        width: 62%;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        border-radius: 5px;
```

Comment:
this would need overflow hidden to i think how is it handle for overflowing content